src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -541,9 +689,778 @@ private bool CheckSensorHandle()
+        public void RecorderOptionSetInt(int recorderOptionE, int value)
+        {
+            if (CheckRecorderOption())
+            { 
+                int error = Interop.SensorRecoder.RecorderOptionSetInt(recorderOption, recorderOptionE, value);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error checking sensor recorder option set int");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderOptionSetInt Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder option is not created");
+            }
+        }
+
+        /// <summary>
+        ///  Sets an integer-type query parameter.
+        /// </summary>
+        /// <param name="recorderQueryE">
+        /// Query parameter.
+        /// </param>
+        /// <param name="value">
+        /// Value.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderQuerySetInt(int recorderQueryE, int value)
+        {
+            if (CheckRecorderQuery())
+            { 
+                int error = Interop.SensorRecoder.RecorderQuerySetInt(recorderQuery, recorderQueryE, value);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error checking sensor recorder query set int");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderQuerySetInt Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder query is not created");
+            }
+        }
+
+        /// <summary>
+        ///  Sets a time-type query parameter.
+        /// </summary>
+        /// <param name="recorderQueryE">
+        /// Query parameter.
+        /// </param>
+        /// <param name="time">
+        /// Time.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderQuerySetTime(int recorderQueryE, int time)
+        {
+            if (CheckRecorderQuery()) { 
+                int error = Interop.SensorRecoder.RecorderQuerySetTime(recorderQuery, recorderQueryE, time);
+                if (error != (int)SensorError.None)
+                {
+                    Log.Error(Globals.LogTag, "Error checking sensor recorder query set time");
+                    throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderQuerySetTime Failed");
+                }
+            }
+            else
+            {
+                Log.Error(Globals.LogTag, "Recorder query is not created");
+            }
+        }
+
+        /// <summary>
+        ///  Queries the recorded data synchronously.
+        /// </summary>
+        /// <param name="type">
+        /// Sensor type.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderReadSync(int type)
```

Comment:
You should return a result becaust it is a sync method
Do not add suffix `Sync` 

```suggestion
        public int RecorderRead(int type)
```

Reply:
Sync suffix removed.